web/docs/project/env-vars.md

```diff
@@ -6,14 +6,14 @@ title: Env Variables
 
 For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you may prefer it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
 
-While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes.
+While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes, and then access them in the code.
```

Comment:
Let's use this chance to get rid of the passive form.
Also, the sentence is a little long and can be broken down into simpler sentences:
```suggestion
Wasp requires you to set certain env vars (e.g., the database connection string, the secrets for social auth).
You can also set your env vars with anything you might need to access from the code (e.g., list some examples for this)
```

Reply:
I get that we don't like passive, but you also removed the "while", therefore changing the flow of the sentence for the worse in my opinion. Also, there might be some other purpose for providing env vars than for accessing them in the code, so you narrowed that down too much.

I wouldn't try to fix the current documentation at this moment, let's do that in another PR if you wish. This is 3 months old PR with relatively small additions, that was created to quickly address the issue user -> we can merge that and get that value, while avoiding the discussion that will now arise from additional trying to improve the docs near by it.